src/engine/parsers/ParsingUtils.java

```diff
@@ -1,8 +1,13 @@
 package engine.parsers;
 
 import engine.GameConstants;
+import gameComponents.DefenseTokens.DefenseToken;
+
+import java.util.Arrays;
 
 public class ParsingUtils {
+        private static String[] VALID_TOKEN_ABBR ={"Br", "Rd", "Ev", "Sc", "Cn", "Sal"};
```

Comment:
Do we want to consider a rename of VALID_TOKEN_ABBR in the event of token types that are not defense tokens? (Note that I cannot think of any of relevance here)

Reply:
Yes, yes we do